make check fails on master

```
$ make check 
/Users/michal/Scylla/scylla-manager/pkg/service/backup/purger.go 41: case taskPolicy.RetentionDays > 0 && t.Before(time.Now().AddDate(0, 0, -taskPolicy.RetentionDays)):
make: *** [Makefile:63: .check-timeutc] Error 1
```

--- a/pkg/service/backup/purger.go
+++ b/pkg/service/backup/purger.go
@@ -38,7 +38,7 @@ func staleTags(manifests []*ManifestInfo, retentionMap RetentionMap) *strset.Set
 			case m.Temporary:
 				tags.Add(m.SnapshotTag)
 			// Tasks can have a Retention policy and a RetentionDays policy so fall through if tag is not too old
-			case taskPolicy.RetentionDays > 0 && t.Before(time.Now().AddDate(0, 0, -taskPolicy.RetentionDays)):
+			case taskPolicy.RetentionDays > 0 && t.Before(timeutc.Now().AddDate(0, 0, -taskPolicy.RetentionDays)):
 				tags.Add(m.SnapshotTag)
 			case taskPolicy.Retention > 0:
 				taskTags.Add(m.SnapshotTag)

--- a/pkg/service/backup/purger_test.go
+++ b/pkg/service/backup/purger_test.go
@@ -3,6 +3,7 @@
 package backup
 
 import (
+	"github.com/scylladb/scylla-manager/pkg/util/timeutc"
 	"testing"
 	"time"
 
@@ -42,7 +43,7 @@ func TestStaleTags(t *testing.T) {
 	manifests = append(manifests, &ManifestInfo{
 		NodeID:      "c",
 		TaskID:      task2,
-		SnapshotTag: SnapshotTagAt(time.Now().AddDate(0, 0, -15)),
+		SnapshotTag: SnapshotTagAt(timeutc.Now().AddDate(0, 0, -15)),
 	})
 
 	// Temporary manifest

